demo.html

```diff
@@ -9,10 +9,26 @@
     <link rel="stylesheet" href="style.css"></head>
 <body>
     <div class="bgc1 clearfix">
-        <h1 class="mhmm mvm"><span class="fgc1">Font Name:</span> openshift-logos-icon <small class="fgc1">(Glyphs:&nbsp;70)</small></h1>
+        <h1 class="mhmm mvm"><span class="fgc1">Font Name:</span> openshift-logos-icon <small class="fgc1">(Glyphs:&nbsp;71)</small></h1>
     </div>
     <div class="clearfix mhl ptl">
         <h1 class="mvm mtn fgc1">Grid Size: 16</h1>
+        <div class="glyph fs1">
```

Comment:
This leaves Cordova rendering [first in the demo page](https://rawgit.com/sedroche/openshift-logos-icon/edf561cd5e05639e125ff05ceb3ad49da1068d4a/demo.html).  Can you update it to reflect it's new order at the end of the list?

Reply:
Done.